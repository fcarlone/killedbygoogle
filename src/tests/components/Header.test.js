import React from 'react';
import { shallow } from 'enzyme';
import Masthead from '../../components/Header';

test('should reander Masthead correctly', () => {
  const wrapper = shallow(<Masthead />);
  expect(wrapper).toMatchSnapshot();
});
