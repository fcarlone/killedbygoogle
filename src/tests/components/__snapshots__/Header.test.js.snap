// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should reander Masthead correctly 1`] = `
<styled.header>
  <styled.img
    alt="Tombstone Icon"
    src="assets/tombstone.svg"
  />
  <styled.h1>
    Killed by Google
  </styled.h1>
</styled.header>
`;
